resolwe_bio/processes/chip_seq/macs2.yml

```diff
@@ -14,7 +14,7 @@
       memory: 16384
       cores: 10
   data_name: "{{ case|sample_name|default('?') }}"
-  version: 4.0.8
+  version: 4.1.0
```

Comment:
This is one of the ugliest processes I've ever seen. Consider rewriting it to Python.

Reply:
That's a lot of work. We can consider rewriting it when/if we introduce major changes to the process.